modules/ROOT/pages/set-credentials-in-studio-to.adoc

```diff
@@ -13,6 +14,9 @@ To enable Anypoint Studio to access resources in your organization, you need to
 If your organization has external identity configured to access Anypoint Platform, click Configure, select External Identity, and type your organization's domain. Talk to your organization administrator if you don't know your organization's domain.
 . Click OK.
 
+When adding a new credential for the first time, Studio notifies you that a new master password has been created. Additionally, Studio prompts you to create two recovery questions to use in case you forget your master password. +
```

Comment:
Who or what is adding a new credential? 
Ditto previous edit questions about identifying the actor (sometimes named "it").